homeassistant/components/lawn_mower/__init__.py

```diff
@@ -0,0 +1,187 @@
+"""The lawn mower integration."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+from datetime import timedelta
+from enum import IntFlag
+from functools import partial
+import logging
+from typing import Any, final
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, ServiceCall
+from homeassistant.helpers.entity import Entity, EntityDescription
+from homeassistant.helpers.entity_component import EntityComponent
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DOMAIN
+
+SCAN_INTERVAL = timedelta(seconds=60)
+
+_LOGGER = logging.getLogger(__name__)
+
+SERVICE_START_MOWING = "start_mowing"
+SERVICE_PAUSE = "pause"
+SERVICE_ENABLE_SCHEDULE = "enable_schedule"
+SERVICE_DISABLE_SCHEDULE = "disable_schedule"
+SERVICE_DOCK = "dock"
+
+ERROR = "error"
```

Comment:
These string constants are never used.

Reply:
Yeah been looking at them for removal after adding Activity. Will remove